datasets/tweets_ar_en_parallel/tweets_ar_en_parallel.py

```diff
@@ -0,0 +1,148 @@
+"""Bilingual Corpus of Arabic-English Parallel Tweets"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import pandas as pd
+
+import datasets
+
+
+_CITATION = """\
+    @inproceedings{Mubarak2020bilingualtweets,
+title={Constructing a Bilingual Corpus of Parallel Tweets},
+author={Mubarak, Hamdy and Hassan, Sabit and Abdelali, Ahmed},
+booktitle={Proceedings of 13th Workshop on Building and Using Comparable Corpora (BUCC)},
+address={Marseille, France},
+year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+    Twitter users often post parallel tweets—tweets that contain the same content but are
+    written in different languages. Parallel tweets can be an important resource for developing
+    machine translation (MT) systems among other natural language processing (NLP) tasks. This
+    resource is a result of a generic method for collecting parallel tweets. Using the method,
+    we compiled a bilingual corpus of English-Arabic parallel tweets and a list of Twitter accounts
+    who post English-Arabic tweets regularly. Additionally, we annotate a subset of Twitter accounts
+    with their countries of origin and topic of interest, which provides insights about the population
+    who post parallel tweets.
+"""
+
+_URL = "https://alt.qcri.org/resources/bilingual_corpus_of_parallel_tweets"
+
+_DATA_URL = "https://alt.qcri.org/wp-content/uploads/2020/08/Bilingual-Corpus-of-Arabic-English-Parallel-Tweets.zip"
+
+
+class ParallelTweetsConfig(datasets.BuilderConfig):
+    """BuilderConfig for Arabic-English Parallel Tweets"""
+
+    def __init__(self, description, data_url, citation, url, **kwrags):
+        """
+        Args:
+            description: `string`, brief description of the dataset
+            data_url: `dictionary`, dict with url for each split of data.
+            citation: `string`, citation for the dataset.
+            url: `string`, url for information about the dataset.
+            **kwrags: keyword arguments frowarded to super
+        """
+        super(ParallelTweetsConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwrags)
+        self.description = description
+        self.data_url = data_url
+        self.citation = citation
+        self.url = url
+
+
+class ParallelTweets(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class TweetsArEnParallel(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `tweets_ar_en_parallel`